### ISSUE TITLE
`alt`-key to focus menu bar can cause issues with alt-commands that open quickpicks and other focus issues

### ISSUE THREAD

**Issue by eamodio** [Microsoft/vscode#63084]
Issue Type: <b>Bug</b>

With the custom menu bar and even with `"window.enableMenuBarMnemonics": false` set, the `alt` key force focus to the menu bar. This can conflicts with `alt` commands that open quickpicks (since it often closes the quickpick when you let go of `alt`) and other focus issues.

Can we just have a setting, or use  `"window.enableMenuBarMnemonics": false` as the signal, to disable `alt` to focus the menu bar all together?

/cc @sbatten 

VS Code version: Code - Insiders 1.30.0-insider (c72cb129f2a0b8117c9b01f3767026041d7d1d31, 2018-11-13T06:19:28.504Z)
OS version: Windows_NT x64 10.0.18277

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz (8 x 4008)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.93GB (21.48GB free)|
|Process Argv|c:\Users\Eric\code\vscode-gitlens|
|Screen Reader|no|
|VM|0%|

</details>

**Comment by sbatten** [comment]
@eamodio can you elaborate on which alt commands? There may be some missing checks in the menu bars alt focus logic

**Comment by sbatten** [comment]
Thanks @eamodio 

The problem seems to be in this case that the click event is eaten before we can detect it in the menu bar so it only notices the alt key. I think we can fix that one, but perhaps we could disable focusing the menu bar in that isn't set to toggle if mnemonics are disabled

**Comment by eamodio** [comment]
> but perhaps we could disable focusing the menu bar in the case that it isn't set to toggle and mnemonics are disabled

That's exactly what I'd want 😄

**Comment by sbatten** [comment]
@Hunter-Github A true ALT+SHIFT should not toggle/focus the menubar. If you are having this issue, please open a separate issue.
